packages/app-project/components/ClassifyPage/components/ConnectWithProject/ConnectWithProjectContainer.spec.js

```diff
@@ -24,7 +24,9 @@ describe('Component > ConnectWithProjectContainer', function () {
     componentWrapper = wrapper.find(ConnectWithProject)
   })
 
-  it('should render without crashing', function () {})
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok
```

Comment:
This is much better than the empty test I've been using - https://github.com/zooniverse/front-end-monorepo/pull/369